app/src/main/res/layout/item_section.xml

```diff
@@ -22,17 +22,30 @@
 
             <TextView
                 android:id="@+id/title"
-                android:layout_width="wrap_content"
+                android:layout_width="276dp"
```

Comment:
Why a fixed size ?

Reply:
Should i make it wrap_content

Reply:
wrap content match parent whatever is better

Reply:
Ok let me solve this